auth0_flutter/README.md

```diff
@@ -330,15 +330,12 @@ Check the [FAQ](FAQ.md) for more information about the alert box that pops up **
 
 ### Common Tasks
 
-#### 📱 Mobile
-
 - [Check for stored credentials](EXAMPLES.md#check-for-stored-credentials) - check if the user is already logged in when your app starts up.
 - [Retrieve stored credentials](EXAMPLES.md#retrieve-stored-credentials) - fetch the user's credentials from the storage, automatically renewing them if they have expired.
-- [Retrieve user information](EXAMPLES.md#retrieve-user-information) - fetch the latest user information from the `/userinfo` endpoint.
 
-#### 🖥️ Web
+#### 📱 Mobile
```

Comment:
These changes are contingent on how we solve the above comments.

Reply:
Now that the web has its own section, I've added a section here for web and linked to it 👍🏻